frontend/lib/Views/AdminPage/GroupsView.dart

```diff
@@ -0,0 +1,159 @@
+            child: Container(
+              padding: const EdgeInsets.only(bottom: 10, top: 5),
+              child: const Text(
+                "User groups",
+                style: TextStyle(fontSize: 25),
+              ),
+            ),
+          ),
+        ),
+      );
+
+  FutureBuilder _buildBody() => FutureBuilder<List<GroupSummary>>(
+      future: widget.repository.getGroupsSummary(),
+      builder: (context, snapshot) {
+        if (snapshot.connectionState == ConnectionState.none ||
+            snapshot.data == null) {
+          return loadingInCenter();
+        }
+        return ChangeNotifierProvider(
+          create: (context) =>
+              GroupListProvider(snapshot.data!, widget.repository),
+          child: Consumer<GroupListProvider>(
+            builder: (context, groupListProvider, _) =>
+                _buildGroupList(groupListProvider, snapshot.data!.length),
+          ),
+        );
+      });
+
+  ListView _buildGroupList(
+          GroupListProvider groupListProvider, int itemCount) =>
+      ListView.builder(
+          itemCount: itemCount,
+          padding: const EdgeInsets.symmetric(horizontal: 20),
+          itemBuilder: (context, i) =>
+              _buildGroupCard(groupListProvider.groupsList[i]));
+
+  Card _buildGroupCard(GroupCard group) => Card(
+        margin: const EdgeInsets.symmetric(vertical: 10, horizontal: 10),
+        shadowColor: Colors.black,
+        shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
+        child: ExpansionTile(
+          onExpansionChanged: (value) {
+            if (value) {
+              setState(() {
+                group.titleColor = Colors.pink;
+              });
+            } else {
+              setState(() {
+                group.titleColor = Colors.black;
+              });
+            }
+          },
+          title: Text(
+            group.name,
+            style: TextStyle(
+                fontFamily: 'SofiaSans',
+                fontSize: 25,
+                color: group.titleColor,
+                fontWeight: FontWeight.w500),
+          ),
+          tilePadding: const EdgeInsets.all(20),
+          iconColor: group.titleColor,
+          childrenPadding: const EdgeInsets.all(0),
+          children: [
+            _buildButtonContainer('Members', group, group.membersButtonColor),
+            _buildButtonContainer(
+                'Endpoints and permissions', group, group.endpointsButtonColor),
+          ],
+        ),
+      );
+
+  Container _buildButtonContainer(
+          String text, GroupCard groupCard, Color color) =>
+      Container(
+        padding: const EdgeInsets.symmetric(vertical: 20, horizontal: 20),
+        child: TextButton(
+          onPressed: () {},
+          onHover: (value) {
+            if (value) {
+              _onHoverButton(groupCard, text, Colors.pink);
+            } else {
+              _onHoverButton(groupCard, text, Colors.black);
+            }
```

Comment:
Hmmm, może zrobimy tak?
```suggestion
final color = value ? Colors.pink : Colors.black;
_onHoverButton(groupCard, text, color);
```